public/assets/styles/login.scss

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css?family=Zilla+Slab');
+@import url('https://fonts.googleapis.com/css?family=Open Sans');
+
+$background-color: #f9f4f4;
+$facebook-color: #3b5998;
+$twitter-color: #00aced;
+$google-color: #f44d4d;
+$white: #ffffff;
+$site-color: #007fff;
+$line-color: #ced4da;
+
+%style-button {
+  border: 0;
+  font-size: 14px;
+  height: 50px;
+}
+
+%input-style {
+  border: 0;
+  border-bottom: 1px solid $line-color;
+  border-radius: 0;
+}
+
+.login-form{
+  background-color: $background-color;
+  border-bottom-left-radius: 10px;
+  border-top-left-radius: 10px;
+  padding: 80px 40px;
+
+  h4{
+    color: $site-color;
+    font-size: 24px;
+  }
+
+  h5{
+    color: $site-color;
+    font-size: 18px;
+    margin-top: 90px;
+  }
+}
+
+.signup{
+  background-color: $site-color;
+  border-bottom-right-radius: 10px;
+  border-top-right-radius: 10px;
+  color: $white;
+  padding: 80px 40px;
+}
+
+.google-btn{
+  @extend %style-button;
+  background-color: $google-color;
+}
+
+.facebook-btn{
+  @extend %style-button;
+  background-color: $facebook-color;
+}
+
+.twitter-btn{
+  @extend %style-button;
+  background-color: $twitter-color;
+}
+
+.twitter-btn:hover{
+  background-color: $twitter-color;
+}
+
+.facebook-btn:hover{
+  background-color: $facebook-color;
+}
+
+.google-btn:hover{
+  background-color: $google-color;
+}
+
+.email-input{
+  @extend %input-style;
+  background-color: transparent;
+}
+
+.email-input:focus {
+  @extend %input-style;
+  background-color: transparent;
+  box-shadow: none;
```

Comment:
does this element come with a default box-shadow?? if it doesn't, then this is not needed